### PR TITLE
feat: add info about rss ai, update dismission, activists and code of conduct

### PR DESCRIPTION
**Pull request by TrickyPie**
- [x] Add info about RSS AI Discord role
- [x] Update info after discussion in RSS AI chat

### CODE REVIEW

File: docs/code-of-conduct.md [hunk 1]
```diff
@@ -64,45 +67,48 @@ The Rolling Scopes School believes our community should be open for everyone.
...
+- sustained disruption of talks, topic channel conversations or events
+- unwelcome sexual attention
 
 The toxic behavour includes:
-* unreasonable repetitive critic of the school, its program, mentors and students
-* rude language outside of random channel
-* insults of other school members in any available communication channels
-* unreasonable conspiracy theory spreaded on school channels
+
+- unreasonable repetitive critic of the school, its program, mentors and students
```
Comment (muannna) [inline docs/code-of-conduct.md]: and supporting this critic

File: docs/dismission.md [hunk 2]
```diff
@@ -1,24 +1,30 @@
...
+
 - Не ожидается, что студенты успеют решить все алгоритмические задачи.
 - Если во время собеседования вы не знаете, как вы решали сданное вами задание ⇒ было списывание ⇒ отчисление.
 - Если во время собеседования вы знаете, как решали задание, но не можете решить заведомо более простую задачу ⇒ было списывание ⇒ отчисление.
 - Хорошо подумайте перед сабмитом чужого кода, стоит ли риск быть отчисленным за 10 баллов.
 
-### Что делать, если вы подозреваете студента в списывании 
+### Что делать, если вы подозреваете студента в списывании
+
 Cообщить модератору курса (личным сообщением)
```
Comment (muannna) [inline docs/dismission.md]: Сообщить модератору курса или RSS AI (личным сообщением)

File: docs/code-of-conduct.md (hunk 1 above)
Comment (TrickyPie) [reply docs/code-of-conduct.md]: done

File: docs/dismission.md (hunk 2 above)
Comment (TrickyPie) [reply docs/dismission.md]: done

**Review by muannna — APPROVED** [review]

**Review by valerydluski — APPROVED** [review]

**Pull request merged**